db/migrate/20230724210704_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Same here a comment belongs to the author Kindly consider adding a reference to the author instead of the user so the relations are set up properly